oaipmhTemplates/dha_RepoObject.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE RDF [
+    <!ENTITY xml "http://www.w3.org/XML/1998/namespace">
+    <!ENTITY xsd "http://www.w3.org/2001/XMLSchema#">
+    <!ENTITY owl "http://www.w3.org/2002/07/owl#">
+    <!ENTITY skos "http://www.w3.org/2004/02/skos/core#">
+    <!ENTITY dc "http://purl.org/dc/elements/1.1/">
+    <!ENTITY dct "http://purl.org/dc/terms/">
+    <!ENTITY foaf "http://xmlns.com/foaf/0.1/">
+    <!ENTITY premis "http://www.loc.gov/premis/rdf/v1#">
+    <!ENTITY dha "https://digital-humanities.at/schema#">
+    <!ENTITY ebucore "http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#">
+]>
+<rdf:RDF xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:xml="&xml;" xmlns:xsd="&xsd;"
+    xmlns:skos="&skos;" xmlns:dc="&dc;" xmlns:dct="&dct;" xmlns:foaf="&foaf;" xmlns:dha="&dha;" xmlns:premis="&premis;">
+
+    <!--
+    ////////////////////////////////////////////////////////////////////////////
+    //
+    // Mapping for RepoObject
+    //
+    ////////////////////////////////////////////////////////////////////////////
+    -->
+
+    <dha:RepoObject> <!-- Q: how do I include rdf:about with URI of the Resource? -->
+
+        <dha:hasMetadataDescription rdf:parseType="Resource">
+            <dc:title count="*" lang="true" val="/acdh:hasTitle"></dc:title>
+            <dc:identifier count="*" val="@acdh:hasIdentifier/acdh:hasTitle"></dc:identifier>
+            <dc:creator count="*" val="@acdh:hasCreator/acdh:hasTitle"></dc:creator>
+            <dc:contributor count="*" val="@acdh:hasContributor/acdh:hasTitle"></dc:contributor>
+        </dha:hasMetadataDescription>
+
+        <dha:basedOn> <!-- Q: is it possible to create as many elements for this property as the values for ^acdh:isSourceOf? -->
```

Comment:
`count="*"`, see [the docs](https://acdh-oeaw.github.io/arche-docs/devdocs/classes/acdhOeaw-arche-oaipmh-metadata-LiveCmdiMetadata.html)

Reply:
ok! But can I create several `<dha:basedOn>` elements with a `rdf:resource` attribute with value = URI of every object of `^acdh:isSourceOf`?

Reply:
> Just if the node has internal structure and the whole internal structure should be repeated, you should use subtemplates. See https://github.com/acdh-oeaw/arche-docker-config/blob/arche/oaipmhTemplates/kulturpool.xml#L49 and https://github.com/acdh-oeaw/arche-docker-config/blob/arche/oaipmhTemplates/kulturpoolHasView.xml

👍